index.d.ts

```diff
@@ -43,6 +43,20 @@ declare namespace OttomanJS {
     validate<T> (modelInstance: ModelInstance<T>, callback: ValidationCallback<T>): void
   }
 
+  interface ModelOptions {
+    index?: IndexDefinition
+    query?: QueriesDefinition
```

Comment:
Please be consistent with the singular form used for IndexDefinition (the terms should be QueryDefinition and Query)

Reply:
oh ok, I did that because the options from ottoman called queries and index